src/main/java/at/hannibal2/skyhanni/SkyHanniMod.kt

```diff
@@ -533,6 +534,7 @@ class SkyHanniMod {
         loadModule(OdgerWaypoint())
         loadModule(TiaRelayHelper())
         loadModule(TiaRelayWaypoints())
+        Relay.entries.forEach { loadModule(it) }
```

Comment:
regex pattern is no event, you dont need to register it here.

Reply:
But repoPatterns need to be initialized in the preinit phase to work. And I do not know how to get it working without loadModule

Reply:
You don't need to load the modules, you just need to make sure that the Relay class is initialized during preinit. You can do that by just accessing the Relay class during either static or instance init of `TiaRelayWaypoints`.